cypress/support/constants/constants.js

```diff
@@ -68,7 +68,7 @@ module.exports = {
   CUCUMBER: 'cucumber',
   CURRENT_APP_ID: 'currentAppId',
   CUSTOM_METHOD_PATH:
-    'https://github.com/rdkcentral/firebolt-certification-suite/blob/main/cypress/support/step_definitions/validations.md#custom',
+    'https://github.com/rdkcentral/firebolt-certification-suite/blob/dev/cypress/fixtures/docs/validations.md#custom',
```

Comment:
this should change when we merge the PR to main ?

Reply:
sure

Reply:
Please change this to "main" before this can be approved.